app/src/main/java/com/voipgrid/vialer/CallActivity.java

```diff
@@ -330,7 +330,7 @@ private void toggleSpeaker() {
     private void toggleMute() {
         mLogger.d("toggleMute");
         mMute = !mMute;
-        updateMicrophoneVolume(mMute ? R.integer.mute_microphone_volume_value : R.integer.unmute_microphone_volume_value);
+        updateMicrophoneVolume(mMute ? getResources().getInteger(R.integer.mute_microphone_volume_value) : getResources().getInteger(R.integer.unmute_microphone_volume_value));
```

Comment:
Get the integer from resources on the previous line so you only have to do it once.

e.g.

int volume =  getResources().getInteger(mMute ?........);
updateMicrophoneVolume(volume);